pkg/testing/integration_test.go

```diff
@@ -98,12 +120,17 @@ func integrationCleanup(t *test.T) {
 	if kontroller.namespace != "" {
 		kontroller.kubeCli.CoreV1().Namespaces().Delete(context.TODO(), kontroller.namespace, metav1.DeleteOptions{})
 	}
+	if kontroller.clusterRoleBinding != nil && kontroller.clusterRoleBinding.Name != "" {
+		kontroller.kubeCli.RbacV1().ClusterRoleBindings().Delete(context.TODO(), kontroller.clusterRoleBinding.Name, metav1.DeleteOptions{})
+	}
+	if kontroller.clusterRole != nil && kontroller.clusterRole.Name != "" {
+		kontroller.kubeCli.RbacV1().ClusterRoles().Delete(context.TODO(), kontroller.clusterRole.Name, metav1.DeleteOptions{})
+	}
 }
 
 const (
 	// appWaitTimeout decides the time we are going to wait for app to be ready
 	appWaitTimeout = 3 * time.Minute
-	controllerSA   = "default"
```

Comment:
Lets just rename it to `kanister-sa` and pass to `getServiceAccount()`?

Reply:
@PrasadG193 I feel like creating separate SA everytime will be better as we are creating cluster role and cluster role bindings with that. Also incase we have failure due to any reason next time run won't fail due to same exist.